Makefile

```diff
@@ -512,7 +512,7 @@ COMMON_SRC = \
             fc/config.c \
             fc/fc_tasks.c \
             fc/fc_msp.c \
-            fc/mw.c \
+            fc/fc_main.c \
```

Comment:
@martinbudden : Some parts of CF are hard to understand, but it's not THAT bad ...